tests/api_tests/test_nph_participant_api.py

```diff
@@ -951,6 +990,7 @@ def tearDown(self):
         self.clear_table_after_test("nph.enrollment_event_type")
         self.clear_table_after_test("nph.participant_ops_data_element")
         self.clear_table_after_test("nph.consent_event")
+        self.clear_table_after_test("nph.consent_event_type")
```

Comment:
Does this need to be here? I dont see this data changing from test to test?

Reply:
the data does not change, but the setUp methods keep adding the same data to the consent_event_type table after every test function. So to avoid having duplicate event types in that table, we added it here.

Reply:
Sounds good